### PR TITLE
Initial german localization

### PR DESCRIPTION
**Pull request by eyecatchup**
Changes proposed in this pull request:

- Add initial localization for German

### CODE REVIEW

File: src/site/content/de/404.njk [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+description: |
+  Seite nicht gefunden
+noindex: true
+---
+
+<div class="w-layout-container--large">
+  <header class="w-page-header">
+    <h1 class="w-page-header__headline w-code">{{ title }}</h1>
+    <p class="w-page-header__copy">
+      Entschuldigung, die Siete konnte nicht gefunden werden.
```
Comment (devnook) [inline src/site/content/de/404.njk:13]: Please sync it with the /en/404.md file (pls note change of the file extension)

File: src/site/content/de/de.11tydata.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    allPaths['accessible'],
+    allPaths['fast'],
+    allPaths['reliable'],
+    allPaths['secure'],
+  ].filter(livePaths);
+
+  return {
+    lang: lang.lang,
+    locale: lang.locale,
+    // translated: 'none',
```
Comment (devnook) [inline src/site/content/de/de.11tydata.js:26]: This is needed for the UI to know that it's not an original post but a translation.

File: src/site/content/de/feed.njk [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  feed:
+    subtitle: Die Zukunft des Webs bauen.
+    filename: feed.xml
+    path: feed/feed.xml
+    url: https://web.dev/feed.xml
+    id: https://web.dev/
+noindex: true
+permalink: "/{{lang}}/feed.xml"
+---
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (devnook) [inline src/site/content/de/feed.njk:18]: Please sync with the /en/ equivalent.

**Review by devnook — CHANGES_REQUESTED** [review]

**Comment by eyecatchup** [comment]
@devnook Updated PR with requested changes.

**Review by devnook — APPROVED** [review]
Thanks @eyecatchup for your pull request! 
All files look fine.
We still need to run a few test on our side, to see how we can handle individual language contributions. I'll unblock and merge this pr as soon as we're ready to start accepting & serving de content on web.dev.

**Comment by robdodson** [comment]
marking this as accepted so the stale bot will ignore it. We're getting close to launching our first test language so I'm hoping we can address this PR in the near future.

**Comment by eyecatchup** [comment]
Nice. As soon as you're ready, I'll gladly translate much more content than I did with this first PR.

**Review by Sifat1998 — APPROVED** [review]

**Comment by conradkirschner** [comment]
@robdodson any status update?

**Comment by robdodson** [comment]
@conradkirschner we're continuing to work on it. We recently moved to firebase hosting which unlocks our ability to have different languages. We're working with a vendor to figure out what our initial set of docs to translate will be.

**Comment by conradkirschner** [comment]
@robdodson so when I understood you correctly, you don't want a community translation? (Then I would unfollow this)

**Comment by robdodson** [comment]
@conradkirschner no, not at the moment. We're going to start with vendor translated content and wait until we get to a good place with that before exploring community translations.

**Comment by robdodson** [comment]
I think I'm going to close this for now, though I absolutely appreciate the work that people put into it. We're actively working on translations as we speak. COVID really slowed our team down but we're trying to ship some initial translations by late March, early April.

**Pull request closed without merging**